batchglm/train/numpy/glm_norm/estimator.py

```diff
@@ -0,0 +1,77 @@
+import logging
+
+import numpy as np
+
+from .external import EstimatorGlm, Model
+from .model_container import ModelContainer
+from ....models.glm_norm.utils import init_par
```

Comment:
add the `init_par` import to `external.py`

Reply:
is that done? It's important to import everything from the external.py so that we have consistent imports and standalone submodules that could easiyl be integrated elsewhere and moved

Reply:
yes!